bundles/org.eclipse.passage.lic.oshi/src/org/eclipse/passage/lic/oshi/NetHardwareAddress.java

```diff
@@ -27,21 +27,7 @@ final class NetHardwareAddress implements Supplier<String> {
 
 	@Override
 	public String get() {
-		byte[] address;
-		try {
-			address = net.getNetworkInterface().getHardwareAddress();
-		} catch (SocketException e) {
-			return null; // mimic null: supports OSHI reading policy
-		}
-		return enlisted(address);
-	}
-
-	private String enlisted(byte[] bytes) {
-		StringBuilder out = new StringBuilder();
-		for (int one : bytes) {
-			out.append(one);
-		}
-		return out.toString();
+		return net.getMacaddr();
```

Comment:
I'm not sure if this is correct.
This class supplies the value for `new NetworkInterface.HwAddress()` and there is already `new NetworkInterface.MacAddress()`, which has the mac-address as value.

A proper replacement for 
```
net.getNetworkInterface().getHardwareAddress()
```
would 
```
net.queryNetworkInterface().getHardwareAddress()
```
If you don't want to use `NetworkInterface.HwAddress` it would probably be better to remove it than just changing its value.

Reply:
Thank you for your finding @HannesWell , I'll have closer look at it.

Reply:
Resolved with #1184